earthpy/tests/test_plot.py

```diff
@@ -88,3 +87,29 @@ def test_single_band_2dims():
     assert arr.ndim == 2
     assert len(fig.axes[0].get_images()) == 1
 
+""" Colorbar Tests """
+
+
+im_arr2 = np.random.randint(10, size=(5, 5))
+fig, ax = plt.subplots(figsize=(5, 5))
+im = ax.imshow(im_arr2,
+               cmap='RdYlGn')
+
+def test_colorbar_height():
+    """Test that the colorbar ax height matches the image axis height"""
+
+    cb = es.colorbar(im)
+
+    try:
+        cb.ax.get_position().height == \
+        im.axes.get_position().height
+    except AssertionError:
+        raise AssertionError("""Colorbar height does not match 
+                             the axes object provided.""")
+
+def test_colorbar_raises_value_error():
+    """Test that a non matbplotlib axis object raises an value error"""
+
+    with pytest.raises(AttributeError):
```

Comment:
This could use the `match` argument to make sure we get the right `AttributeError`, e.g., 

```suggestion
    with pytest.raises(AttributeError, match="requires a matplotlib"):
```

See https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions for more info